lineage/states/StateDistribution.py

```diff
@@ -86,7 +86,7 @@ def __repr__(self):
         """
         Method to print out a state distribution object.
         """
-        return "State object w/ parameters: {}, {}, {}.".format(self.bern_p, self.gamma_a, self.gamma_scale)
+        return f"State object w/ parameters: {self.params}."
```

Comment:
Did you mean to have this change? What is it?

Reply:
That's from issue #331. Shakthi just wanted to change it to an f-string and call self.params instead of listing all of the parameters